### PR TITLE
Create session page

### PR DESCRIPTION
**Pull request by mko02**

### CODE REVIEW

File: Guess-E/Guess_E.entitlements
Comment (atlant04) [inline Guess-E/Guess_E.entitlements:1]: Is this file necessary?

**Review by atlant04 — COMMENTED** [review]
looks good, there is a change to entitlements, is that necessary?

**Review by azooz2003-bit — APPROVED** [review]

**Comment by azooz2003-bit** [comment]
@mko02  Some logic mistakes when removing a player, but we can fix that during the logic phase. Approved.
